packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -402,4 +402,137 @@ void main() {
 
     handle.dispose();
   });
+
+  testWidgets('Toggle radio panel list when clicking on header',  (WidgetTester tester) async {
+    const Key firstPanelKey = Key('firstPanelKey');
+    const Key secondPanelKey = Key('secondPanelKey');
+
+    final List<ExpansionPanel> _demoItemsRadio = <ExpansionPanelRadio>[
+      ExpansionPanelRadio(
+        headerBuilder: (BuildContext context, bool isExpanded) {
+          return Text(isExpanded ? 'B' : 'A', key: firstPanelKey,);
+        },
+        body: const SizedBox(height: 100.0),
+        value: 0,
+        canTapOnHeader: true,
+      ),
+      ExpansionPanelRadio(
+        headerBuilder: (BuildContext context, bool isExpanded) {
+          return Text(isExpanded ? 'D' : 'C', key: secondPanelKey,);
```

Comment:
If single line, no need for trailing comma. Alternatively, make this multi-line.